dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -677,15 +679,16 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert workshop.virtual?
   end
 
-  test 'cannot update a workshop to be virtual without suppressed email' do
+  test 'can update a workshop to be virtual without suppressed email' do
```

Comment:
I liked how in another test you updated like this that you noted this test exists because it's a change to a previous behavior, could you add something like that here?

Reply:
yep!